pymongo/client_session.py

```diff
@@ -952,6 +952,8 @@ def _update_read_concern(self, cmd, sock_info):
             rc['level'] = 'snapshot'
             if self._snapshot_time is not None:
                 rc['atClusterTime'] = self._snapshot_time
+    def __copy__(self):
```

Comment:
There should be a blank line before this method.

Reply:
Done.

Reply:
I don't see any changes. Did you forget to push?

Reply:
Yeah I git stashed them by accident. Restored it and pushed it now.